src/three-components/SmoothControls.ts

```diff
@@ -597,4 +649,33 @@ export class SmoothControls extends EventDispatcher {
       event.preventDefault();
     }
   }
+
+  [$calculateOverSphericals]() {
+    const {
+      minimumAzimuthalAngle,
+      maximumAzimuthalAngle,
+      minimumPolarAngle,
+      maximumPolarAngle,
+      minimumRadius,
+      maximumRadius,
+      overRotateAmount
+    } = this[$options];
+
+    const minOverRotatePolar = (0 - minimumPolarAngle!) * overRotateAmount! + minimumPolarAngle!;
```

Comment:
These calculations are simple enough that I don't think they warrant a whole extra method and two members. Plus I would not have intuited this function from the overRotateAmount name or description. Perhaps we should change overRotateAmount to specify the number of radians of extra rotation, which would both simplify the description and the calculations.